nautobot/core/celery/backends.py

```diff
@@ -10,18 +10,9 @@ class NautobotDatabaseBackend(DatabaseBackend):
 
     TaskModel = JobResult
 
-    def encode_content(self, data):
-        """Pass through encoding since we're storing as JSON explicitly."""
-        return "application/x-nautobot-json", "utf-8", data
-
-    def decode_content(self, obj, content):
-        """Pass through decoding since we're storing as JSON explicitly."""
-        return content
-
```

Comment:
Removed since we're handling `task_args` and `task_kwargs` below. We may need to bypass again if we implement #3229 but we'll need to see if this negatively impacts storing the group result when running advanced workflows.

Reply:
We need to define `encode_content` and `decode_content` because the default implementation tries to access `content_encoding` on the `TaskResult` but we're throwing that away on the `JobResult`